nemo/core/callbacks.py

```diff
@@ -449,6 +449,8 @@ def on_action_start(self):
             if self._wandb_name is not None or self._wandb_project is not None:
                 if _WANDB_AVAILABLE and wandb.run is None:
                     wandb.init(name=self._wandb_name, project=self._wandb_project)
+                elif wandb.run is not None:
```

Comment:
If _WANDB_AVAILABLE is false, then that means wandb is a None object, therefore wandb.run will throw an error.  Needs to check for the above flag prior to any operation on wandb

Reply:
thanks! fixed